.github/workflows/iroha2-dev-pr.yml

```diff
@@ -102,14 +102,16 @@ jobs:
         run: |
           mv ${{ env.WASM_TARGET_DIR }}/libs ${{ env.DEFAULTS_DIR }}/libs
           mv ${{ env.DEFAULTS_DIR }}/libs/default_executor.wasm ${{ env.DEFAULTS_DIR }}/executor.wasm
-      - name: Install irohad
-        run: which irohad || cargo install --path crates/irohad --locked
```

Comment:
By the way, what is the possible cases that `irohad` is found by `which` command here @0x009922  ?

Reply:
As far as I remember, it was about caching and the possibility that `irohad` is already available. Though I don't understand how that could be possible.